app/components/Admin/NaicsCodeList.tsx

```diff
@@ -0,0 +1,30 @@
+import Link from 'next/link';
```

Comment:
Might it be better to name this file/component something more specific like `SelectableNaicsCodeList`? A few times I wondered if it was at all related or used in the list of NAICS codes at `/admin/naics-codes`.